core/src/test/java/apoc/trigger/TriggerClusterRoutingTest.java

```diff
@@ -0,0 +1,71 @@
+package apoc.trigger;
+
+import apoc.util.Neo4jContainerExtension;
+import apoc.util.TestContainerUtil;
+import apoc.util.TestcontainersCausalCluster;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.neo4j.driver.types.Node;
+import org.neo4j.internal.helpers.collection.MapUtil;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static apoc.trigger.TriggerDeprecatedProcedures.SYS_NON_LEADER_ERROR;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class TriggerClusterRoutingTest {
+
+    private static TestcontainersCausalCluster cluster;
+
+    @BeforeClass
+    public static void setupCluster() {
+        cluster = TestContainerUtil
+                .createEnterpriseCluster(3, 1, Collections.emptyMap(), MapUtil.stringMap(
+                        "NEO4J_dbms_routing_enabled", "true",
+                        "apoc.trigger.enabled", "true"
+                ));
+    }
+
+    @AfterClass
+    public static void bringDownCluster() {
+        if (cluster != null) {
+            cluster.close();
+        }
+    }
+
```

Comment:
I think it would be nice to add some of the new procedures to this file too. 

Of particular interest I think would be:
- making sure that a session against "system" can install/drop triggers
- making sure that a session against "neo4j" can't install/drop triggers
- fabric tests perhaps, so that we can see if that works? I'm not sure you can do `USE system` in Fabric for example 🤔 And I'm not SystemOnlyProcedure annotation has been implemented for Neo4j yet

Reply:
Personally, I think it is probably best to wait for the `@SystemOnlyProcedure` annotation before doing these tests? Just to be sure i create sensible tests.

Reply:
For the Fabric tests then I would be happy to wait.

But I'm not sure we have to wait if want to test that we can't install triggers against "neo4j" or whether we want to test that dropping triggers works. You can use apoc.trigger.list and apoc.trigger.show to confirm whether a trigger was dropped or not.

Reply:
Added test against "neo4j" session.